scripts/get_das_info.py

```diff
@@ -181,6 +185,6 @@ def print_das_info(
 if __name__ == "__main__":
     parser = ArgumentParser()
     parser.add_argument("-d", "--dataset", dest="dataset", nargs="+", help="das name")
-    parser.add_argument("-f", "--function", dest="function", help="function that converts info into code")
+    parser.add_argument("-c", "--convert", dest="convert", help="function that converts info into code")
```

Comment:
Just repeating the suggestion to add `default` and `choices` here in case you missed it ;)
```suggestion
parser.add_argument("-c", "--convert", dest="convert", default="default", choices=list(convert_functions), help="function that converts info into code")
``

Reply:
Hm, I see now that the suggestion is outdated. I probably started the review before your latest comment came is, sorry about that. In the latest head it looks good, so I'll go ahead and resolve this.